base/MAPL_Generic.F90

```diff
@@ -4673,7 +4674,9 @@ recursive integer function MAPL_AddChildFromMeta(META, NAME, GRID, &
   call CHILD_META%t_profiler%start(__RC__)
   call CHILD_META%t_profiler%start('SetService',__RC__)
   gridcomp => META%GET_CHILD_GRIDCOMP(I)
-  call ESMF_GridCompSetServices ( gridcomp, SS, RC=status )
+  call ESMF_GridCompSetServices ( gridcomp, SS, userRC=SS_STATUS, RC=status )
+  _VERIFY(STATUS)
+  _VERIFY(SS_STATUS)
```

Comment:
@Gvilla1000-nasa Does your refactoring include something like this? (proper error handling for if the user set services fails)

Reply:
Hi @LiamBindle. It does not. My refactoring was simply moving files from one directory to another and making sure dependencies didn't break. In that essence what happened was that `base/MAPL_Generic.F90` simply became `generic/MAPL_Generic.F90`. The changes done here would simply need to be reapplied to that file. Only change I did to the new MAPL_Generic.F90 was to add `use mapl_MaplGrid` at the top.